### ISSUE TITLE
toFile is not recognized as a key

### ISSUE THREAD

**Issue by joychugh** [jrudolph/sbt-dependency-graph#167]
```
sbt:podcast-realtime-subject-extraction-parent> dependencyTree:toFile
[error] Not a valid key: toFile (similar: testFilter, compile, compilers)
[error] dependencyTree:toFile
[error]                      ^
sbt:podcast-realtime-subject-extraction-parent> dependencyTree::toFile
[error] Not a valid key: toFile (similar: testFilter, compile, compilers)
[error] dependencyTree::toFile
[error] 
``` 
sbt version 1.1.0 
dependency graph version 0.9.2 
scala version 2.11.11
os: Mac OS 10.13.6 
Plugin installed in `~/.sbt/1.0/plugins/plugins.sbt` 
`addSbtPlugin("net.virtual-void" % "sbt-dependency-graph" % "0.9.2")`

**Comment by jrudolph** [comment]
The feature has not been released ;)

**Comment by jaceklaskowski** [comment]
A solution is to tag the repo (e.g. `v0.9.2-local`) and `sbt publishLocal`. That will publish a local build of the plugin with the version that matches the tag (as sbt-dependency-graph uses [sbt-dynver](https://github.com/dwijnand/sbt-dynver) sbt plugin).

A much better solution would be to have an official release.

**Comment by shijinkui** [comment]
:(
Why the non-released feature show in readme.
Waste time for debug

**Comment by jrudolph** [comment]
Yeah, sorry, that's really painful. I had a bit more time/energy in last summer and started preparing the new release before other things took over again. I try to get a release out soon.

**Comment by nivekastoreth** [comment]
Any idea when a version including this will be released? I sadly just realized I ran into this ticket when I got to the point I was debugging decompiled sbt code (I didn't think I'd be getting that far when I started, otherwise I'd have set up a proper debugging env) and realized the code that was running didn't match what I was expecting to see based on this `master` repo =/

**Comment by anilkumarmyla** [comment]
Looking for this feature as well, seems the code in master has the feature ready and just needs a release. Let us know if you need any help pushing a new release out

**Comment by jrudolph** [comment]
Released with 0.10.0-RC1.
